components/tikv_util/src/future_pool/mod.rs

```diff
@@ -280,7 +280,7 @@ mod tests {
         assert!(rx.try_recv().is_err());
 
         spawn_future_and_wait(&pool, TICK_INTERVAL / 20);
-        assert_eq!(rx.recv_timeout(Duration::from_micros(50)).unwrap(), 0);
+        assert_eq!(rx.recv_timeout(Duration::from_micros(100)).unwrap(), 0);
```

Comment:
`TICK_INTERVAL / 20` is 50ms. I think it's ok to increase it to 10ms to make it more stable.

Reply:
OK.

Reply:
Could we extract a variable instead of using hard-coded values? Such as `Duration::from_micros(100)`

Reply:
Why is this assertion flaky? If it is flaky at x ms, it is likely to just be less flaky, not stable at any other value of x. Unless, the timeout is really too short, I'd like to fix the root cause - maybe we need to run this test without other tests in parallel, or maybe it needs to be run with a different executor or something?

Reply:
The root is that the tick method is invoked in the thread pool, just after making the future ready. So running the tick method and doing such an assertion are in parallel.
I have another idea. Maybe we can do these assertions inside futures spawned to the thread pool. As the pool has only one thread, we can make sure those futures run after the tick method.